### ISSUE TITLE
Create action should be able to return the newly selected value

### ISSUE THREAD

**Issue by allancaffee** [cibernox/ember-power-select-with-create#38]
It would be really helpful if the `oncreate` action could directly signal `ember-power-select` to use the newly created model as the selected value. I could see this being accomplished by either

1. Resolving a promise returned by the action and updating the selection with the result
2. Passing in the `ember-power-select` `publicAPI` object similarly to how the core library does with all actions. I believe would allow callers to update the selection explicitly.

I see that some of the examples will explicitly update a given field `selectedCountry` in the `oncreate` hook, but the hook itself does not always have all of the required context readily available. For instance if you're building up a list of models it's a hassle to try and get at the context in order to update the correct field.

**Comment by allancaffee** [comment]
I just realized that there's a proposed fix for this in #25. Also I was able to accomplish what I needed to by wrapping my action in another action to curry the model object I was trying to set the field on. Here's a boiled down example:

```js
{{#power-select-with-create
  search=(action "searchField")
  selected=myModel.field
  onchange=(action (mut myModel.field))
  oncreate=(action (action "handleFieldCreation" myModel))
  as |m|
}}
  {{m.name}}
{{/power-select-with-create}}
```

That allows `handleFieldCreation` to update the field explicitly after creating the new model.

**Comment by cibernox** [comment]
> Passing in the ember-power-select publicAPI object similarly to how the core library does with all actions. I believe would allow callers to update the selection explicitly.

@allancaffee Is that not the case? If not, that seems like a bug to me. All actions should receive the publicAPI of the component as last or penultimate argument

**Comment by cibernox** [comment]
@allancaffee I've just merged a PR that might have fixed the issue and actions should receive the publicAPI of the component as second argument. I haven't released a new version yet, but it's in master.

**Comment by felixkiss** [comment]
Closed by #25
